.config

```diff
@@ -0,0 +1,104 @@
+# SPDX-FileCopyrightText: 2025 ANSSI
```

Comment:
remove that file.
Instead -> add a defconfig file in the configs/examples/boardname_defconfig
it should only hold configuration entries that are not default one.
In this very path, just do `savedefconfig --out your_defconfig_file` and add it in gconf instead